android-sdk/src/main/java/com/checkout/android_sdk/logging/FramesLoggingEventDataProvider.kt

```diff
@@ -0,0 +1,19 @@
+package com.checkout.android_sdk.logging
+
+import com.checkout.android_sdk.Utils.Environment
+import com.checkout.eventlogger.domain.model.MonitoringLevel
+
+object FramesLoggingEventDataProvider {
+
+    fun logCheckoutApiClientInitialisedEvent(mEnvironment: Environment): FramesLoggingEvent {
+        val eventData = mapOf(
+            CheckoutApiClientInitEventAttribute.environment to mEnvironment
```

Comment:
`mEnvironment` would refer to a member variable. Shouldn't this just be `environment`?

Reply:
Sure.